spec/rubocop/cop/lint/empty_conditional_body_spec.rb

```diff
@@ -27,6 +29,64 @@
       ^^^^^^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
       end
     RUBY
+
+    expect_correction(<<~RUBY)
+      if condition
+        do_something
+      end
+    RUBY
+  end
+
+  it 'registers an offense for missing `if` body with `else`' do
+    expect_offense(<<~RUBY)
+      if condition
+      ^^^^^^^^^^^^ Avoid `if` branches without a body.
+      else
+        do_something
+      end
+    RUBY
+
+    expect_correction(<<~RUBY)
+      unless condition
+        do_something
+      end
+    RUBY
+  end
+
+  it 'registers an offense for missing `unless` body with `else`' do
+    expect_offense(<<~RUBY)
+      unless condition
+      ^^^^^^^^^^^^^^^^ Avoid `unless` branches without a body.
+      else
+        do_something
+      end
+    RUBY
+
+    expect_correction(<<~RUBY)
+      if condition
+        do_something
+      end
+    RUBY
+  end
+
+  it 'registers an offense for missing `if` body with `elsif`' do
+    expect_offense(<<~RUBY)
+      if condition
+      ^^^^^^^^^^^^ Avoid `if` branches without a body.
+      elsif other_condition
+        do_something
+      elsif another_condition
+        do_something_else
+      end
+    RUBY
+
+    expect_correction(<<~RUBY)
+      if other_condition
+        do_something
+      elsif another_condition
+        do_something_else
+      end
+    RUBY
```

Comment:
@dvandersluis this correction doesn't have the same semantics. To get the same meaning, it should be:
```ruby
if !condition && other_condition
  do_something
elsif !condition && another_condition
  do_something_else
end
```
I don't know if this autocorrection was considered safe before, but it definitely isn't safe or correct now.